apps/music/test/unit/mock_lazy_loader.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+/* exported MockLazyLoader */
+
+// Our MockLazyLoader is better than the shared one because we actually use a
+// Promise for load(). However, trying to do that with the shared version breaks
+// all the system app's tests. Oh well.
+var MockLazyLoader = {
+  load: function(fileArray, callback) {
+    if (callback) {
+      callback();
+    } else {
+      return Promise.resolve();
+    }
+  },
+
+  getJSON: function(file) {
+    return Promise.resolve({});
+  }
+};
```

Comment:
@jimporter Since mock_lazy_loader.js could be used by any app's tests, it will be nice to put them in the /shared/test/ folder, though it's a Promised-based version but people can benefit from this.